public/static/locales/cy/listings.json

```diff
@@ -72,5 +73,15 @@
     "of": " o",
     "firms": "o gwmnïau",
     "order": "Cwmnïau wedi'u cyflwyno mewn unrhyw drefn benodol"
+  },
+  "toolTips": {
+    "age": "Nid yw rhai cwmnïau'n cynnig yswiriant uwchlaw oedran penodol.  The length of trip, whether the trip is a cruise and the destination may affect this.",
```

Comment:
Part of this copy is in English. Please can we add the correct copy? 🏴󠁧󠁢󠁷󠁬󠁳󠁿  😄

Reply:
The thing is the welsh copy has not been provided yet. But if you like, i can use google translations for now

Reply:
No, it's preferable not to use Google for the translations 😱 . 
From what I can see all the Welsh text is in place in the translation file. If that has not been supplied by our agency then I suggest we do not proceed with this card until we have the complete copy in both languages.

Reply:
Fixed

Reply:
They are just placeholders until the agency provided the translations. I dont even think thats the final english copy

Reply:
I would prefer not to use placeholder copy for the Welsh text but leave it blank or with a message that we are awaiting the copy. Otherwise we could assume that is the final copy and deploy it. Strictly speaking this card should not be deployed until all the copy - in both languages - is in place but it would be acceptable to have a further card to add this later.

Reply:
Placeholders have been removed